### ISSUE TITLE
Feature request - ExerciseViewer split view: map/graph

### ISSUE THREAD

**Issue by batzkass** [ssaring/sportstracker#207]
Hi, and thanks for this software, its really handy and helpful.
There is a double feature which is really nice, proposed by other training software but seems to lack here.
It is:
- A combo view of exercises that exposes graph (with altitude for example) and the track on a map at the same time. This way, the slider that allows to "follow" the track on the map would also move a vertical line on the graph. Without that, its almost impossible to make a relation between the position you were on the map and the slope or the speed you were riding.
- ~~In the same fashion as [openrunner](https://sites.google.com/site/wwyrvtt/_/rsrc/1491983029070/2017/Sans%20titre-1%20copie.jpg) do, it would be really helpful to show the slope (in %) with a color under the altitude graph. For this, (lets say 5) ranges of slopes could be represented in 5 different color darkness, for instance (0-5% ; 5-10% ; 10-13% ; 13-15%; 15-100%).~~ => Already done, I will push a new branch tomorrow for a future merge. Here is a preview layout of this:
![SlopeView](https://user-images.githubusercontent.com/1590208/57257217-10564400-7059-11e9-9fc9-15b3db1ca83f.png)

**Comment by ssaring** [comment]
These are 2 quite interesting feature requests. 

The first one sounds very helpful and I like it a lot on other platforms, e.g. when planning tracks in Komoot. So it makes a lot of sense to add it to the ExerciseViewer too, at least for the altitude data.

The second sounds promising too, I've already seen this on other platforms. e.g. Komoot. Although this is somehow related to the first feature, it would be better to track this request separately. This can be done later on when the first is finished.
Could you please create a new issue for it and adjust this feature request?

**Comment by batzkass** [comment]
Ok, so I edited my initial message, you should have a look :-P, let's focus on the first feature here then.

I think that the better way to achieve this would be to merge the last two tabs as it would be totally generic. The only caveat I see is the window size that will need to be larger, but anyway it's easy to toggle fullscreen.

**Comment by ssaring** [comment]
This feature request has been solved by pull request #210 which adds an altitude chart below the map to the Track Panel if altitude data is available.

I agree, it sounds interesting to merge the Diagram and the Track panel into one single panel. I've tried several layouts, but none of the results looked good. And it was quite unusable, especially on laptop screens.
So I've added just the altitude chart with a quite small height below the map, this is the most important chart the user wants to see together with the map. And it also displays the current track position by using a marker line. I think that's a good compromise.
